README.md

```diff
@@ -89,18 +89,26 @@ optional arguments:
 ```
 * To install package dependencies using pip, run:
 ```
+    * If you logged in as root user *
     $ pip install requests
     $ pip install click
+
+    * If you logged in as normal user *
+    $ pip install --user requests
+    $ pip install --user click
+
 ```
 * We recommend to create and activate a virtualenv first:
 ```
-	$ cd readit/
 
   	$ virtualenv -p /usr/bin/python virt
```

Comment:
we do  not require these changes because
- we need an individual virtual environment for project `readit`
- creating a virtual environment inside `readit` folder will help
-  keep old instructions as it is in README.md